src/test/groovy/com/palantir/gradle/externalpublish/ExternalPublishRootPluginIntegrationSpec.groovy

```diff
@@ -527,6 +535,22 @@ class ExternalPublishRootPluginIntegrationSpec extends IntegrationSpec {
         stdout.contains(':gradle-plugin:publishPlugins SKIPPED')
     }
 
+    def 'fixes gradle#26091' () {
+        setup:
+        publishGradlePlugin()
+
+        when:
+        ExecutionResult result = runSuccessfullyWithSigning('-P__TESTING_CIRCLE_TAG=tag', 'publishToMavenLocal', "--info", "--warning-mode=fail")
```

Comment:
"--warning-mode=fail" making sure there are no gradle warnings anymore.

Reply:
I think it's be better to just test with Gradle 8 using `gradleVersion = '8.6'`.